File: u2-bruteforce-sorting/bubblesort.py
#bubblesort program in python
def bubblesort(list):
    if(len(list)==0):
        return -1
    else:
        print(len(list))
        for i in range(len(list)-1):
            for j in range(len(list)-1-i):
                if(list[j]>list[j+1]):
                    #swap if lesser
                    list[j],list[j+1]=list[j+1],list[j]
        return list

#print(bubblesort([9,5,6,13,4,10]))
ls=[]
n=int(input("Enter the number of elements: "))
def enterele(list):
    for i in range(n):
        list.append(int(input("Enter the element: ")))
    return list
enterele(ls)
print(bubblesort(ls))            
        

File: u2-bruteforce-sorting/bubblesort.c
//bubblesort program in c
#include<stdio.h>
#include<stdlib.h>

void enterele(int *a,int n)
{
    printf("Enter the elements:");
    for(int i=0;i<n;i++)
    {
        scanf("%d",&a[i]);
    }
}
void swapper(int *a,int *b)
{
    int temp;
    temp=*a;
    *a=*b;
    *b=temp;
}
void bubblesort(int *a,int n)
{
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-i-1;j++)
        {
            if(a[j]>a[j+1])
            {
                swapper(&a[j],&a[j+1]);
            }
        }
    }
    
}
void printarray(int *a,int n)
{
    for(int i=0;i<n;i++)
    {
        printf("%d) %d\n",i+1,a[i]);
    }
}
void main()
{
    printf("Enter the size of the array:");
    int n;
    scanf("%d",&n);
    int a[n];
    enterele(a,n);
    bubblesort(a,n);
    printarray(a,n);
}